ansible/library/bgp_facts.py

```diff
@@ -76,6 +76,13 @@ def run(self):
         """
             Main method of the class
         """
+
+        # Check if bgp is enabled as a feature, and if not return facts with empty bgp_neighbors.
+        rc, self.out, err = self.module.run_command("show feature status bgp", executable='/bin/bash', use_unsafe_shell=True)
+        regex_bgp = re.compile(r'bgp\s+disabled')
```

Comment:
Can you fix precommit errors:
ansible/library/bgp_facts.py:81:121: E501 line too long (125 > 120 characters)
ansible/library/bgp_facts.py:82:21: F405 're' may be undefined, or defined from star imports: ansible.module_utils.basic

Reply:
@SuvarnaMeenakshi , I have fixed the above two pre-commit-check issues. Please take a look.